frame/erc1155/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "pallet-erc1155"
```

Comment:
ERC are Ethereum thing, which makes them a poor name for Substrate pallets.
This pallet at best implements an ERC1155-like interface and for many non eth devs this name is just confusing.

Reply:
I completely agree with that! We started to brainstorm a new generic name recently but looks like it's the hardest part here :D 
Any suggestions?

Reply:
pallet-fraction-nft?